app/models/manageiq/providers/ibm_cloud/vpc/manager_mixin.rb

```diff
@@ -19,11 +19,21 @@ def connect(options = {})
   end
 
   # Same as calling connect.
-  # @param _auth_type [nil] Not used
+  # @param _auth_type [nil]
   # @param options [Hash] Connection options.
-  # @return [ManageIQ::Providers::IbmCloud::CloudTools::Vpc]
+  # @return [ManageIQ::Providers::IbmCloud::CloudTools::Vpc] or [Boolean]
   def verify_credentials(_auth_type = nil, options = {})
-    connect(options).authenticator
+    case _auth_type&.to_sym
+    when :events
+      verify_events_credentials
+    else
+      connect(options).authenticator
```

Comment:
For symmetry maybe we can pull this out into a `verify_default_credentials` method